src/pages/AuthPage/AuthPage.style.jsx

```diff
@@ -0,0 +1,228 @@
+import styled from "styled-components";
+
+export const Auth = styled.article`
+    color: #858598;
+    font-family: 'Montserrat-Regular';
+    position: relative;
+`;
+
+export const AuthBody = styled.article`
+    display: flex;
+    justify-content: space-between;
+    margin: 20px 123px 21px 16px;
+
+    @media (max-width: 1150px) {
+        margin-right: 50px;
+    }
+
+    @media (max-width: 1060px) {
+        margin-right: 16px;
+    }
+
+    @media (max-width: 800px) {
+        flex-direction: column-reverse;
+        margin-left: 0;
+        margin-right: 0;
+    }
+
+    @media (max-width: 320px) {
+        margin-bottom: 0;
+    }
+`;
+
+export const Collage = styled.div`
+    position: relative;
+    width: 580px;
+    height: 745px;
+
+    @media (max-width: 800px) {
+        width: 100%;
+        height: 239px;
+    }
+`;
+
+export const CollageImageWrapper = styled.div`
+    position: absolute;
+    filter: drop-shadow(4px 6px 15px rgba(0, 0, 0, 0.1));
+
+    :nth-child(1){
+        width: 372px;
+        height: 306px;
+    }
+
+    :nth-child(2){
+        width: 279px;
+        height: 264px;
+        top: 170px;
+        right: 80px;
+    }
+
+    :nth-child(3){
+        width: 276px;
+        height: 219px;
+        top: 326px;
+    }
+
+    :nth-child(4){
+        width: 359px;
+        height: 290px;
+        bottom: 0;
```

Comment:
such selectors make sense if you have lots of items and can reuse logic.
For example `:nth-child(2)` will add styles for every second child, but you need this styles for one element.
I think we can get better code if we will have different wrappers with different styles for each wrapper